common/sai_client/sai_thrift_client/sai_thrift_utils.py

```diff
@@ -74,7 +74,11 @@ def convert_value_to_thrift(value, attr_name=None, value_type=None):
                 actual_value = getattr(sai_headers, value, None)
                 if actual_value != None:
                     return actual_value
-            return 0 if value == '' else int(value, 0)
+            try:
```

Comment:
Why have you put this check here? Is it Thrift specific? Will Redis RPC work in case the value will be an integer but not string?...